src/ui.popup.js

```diff
@@ -67,19 +67,19 @@ var _ = Mavo.UI.Popup = $.Class({
 		if (this.editor.matches("select")) {
 			this.editor.size = Math.min(10, this.editor.children.length);
 		}
+		var _popup = this;
+		this.hideCallback = evt => {
+			if (!_popup.element.contains(evt.target) && !_popup.primitive.element.contains(evt.target)) {
+				_popup.hide();
+			}
+		};
```

Comment:
Hey, thanks!
You don't need `_popup`, arrow functions have no `this` binding, so they inherit that of the containing scope. So you can just use `this` in it, just like it was using below.

Reply:
Thank you for being so open minded to PR.

I have changed it back to arrow function and verified the code works on my local dev server. I probably had a typo somewhere else and the dev console attributed the error to the arrow function.